frontend/src/components/Details/Details.jsx

```diff
@@ -0,0 +1,51 @@
+import { useDispatch, useSelector } from 'react-redux'
+import './Details.css'
+import Loading from './Loading'
```

Comment:
[REQUIRED]the file Loading is not found when running the app.
Kindly see how to fix that so the page can properly load.

it's displaying this message bellow

![image](https://github.com/sajibarafatsiddiqui/final-capstone-project/assets/68948316/1a772a9c-b0ef-4aa4-b1a6-6a1f84c9dfd6)